lib/mosaic-geomath/src/main/java/org/eclipse/mosaic/lib/spatial/QuadTreeTraversal.java

```diff
@@ -74,6 +74,11 @@ private static <T> void selectInArea(QuadTree.TreeNode node, BoundingBox area, P
         }
     }
 
+    private static boolean containsOnPlane(BoundingBox area, Vector3d p) {
```

Comment:
Did this cause issues before, or is this change for performance reasons?

Reply:
Yes, with phabmacs we could not found vehicles in the index, cause the `y`-coordinate of the bounding-box was `0` and the y-coordinates of the vehicles was `0.5`.

Reply:
Maybe this should be a method of `BoundingBox` itself?

Reply:
The BoundingBox works in 3d-space, instead we would need a BoundingBox2d. But then we also should not store Vector3d for the positions. The QuadTree is only 2-dimensional on the XZ-plane, as stated in the class comment, but uses Vector3d. This indeed not clean, but changing the API would also require Phabmacs to change their code, which I want to avoid. We could discuss this with @fabmax later.